lib/new-package.js

```diff
@@ -455,11 +461,14 @@ function packageJson(options, packageName) {
       'update-examples': 'node maintenance/update-examples-from-preview.js',
       postinstall: 'elm-tooling install'
     },
-    dependencies: {
+    engines: {
+      node: '>=14.21.3'
+    },
+    devDependencies: {
```

Comment:
I'd keep these as dependencies. Elm packages that use `package.json` tend to have these as regular dependencies. Though I'm not sure that one is better than the other :shrug:

Reply:
I'd argue they're better as dependencies, but I'd meant to move these to another PR anyway.